apps/modernization-ui/src/apps/page-builder/page/management/edit/add-question/search/question-search.module.scss

```diff
@@ -0,0 +1,51 @@
+@use 'styles/borders';
+@use 'styles/colors';
+
+.content {
+    padding: 0 1.5rem;
+    display: flex;
+    min-height: 100px;
```

Comment:
px to rem

Reply:
good catch. looks like i can remove this min-height completely